src/js/classes/Table.js

```diff
@@ -131,7 +131,7 @@ class BarGraphCell extends Cell {
     this.element.appendChild(bar);
     // add the vertical line denoting the average
     const averageLine = document.createElement("div");
-    averageLine.className = "bar-average-line green";
+    averageLine.className = "bar-average-line white";
```

Comment:
This should be `white-race`, right? That seems like what you've been using for the others

Reply:
Lol the racial color is so confusing! It's supposed to be referring from this line in figma, but it's actually more gray, so I'll call it that.
<img width="247" alt="Screen Shot 2022-08-29 at 20 25 53" src="https://user-images.githubusercontent.com/4998601/187321680-2dfef5f0-a82c-4b9e-9e65-b44f00b8bf42.png">